config/locales/fr.yml

```diff
@@ -50,13 +50,13 @@ fr:
       no_subscriptions: Vous n'avez pas encore d'abonnements. Visitez %{gem_link} pour vous y abonner !
   email_confirmations:
     create:
-      promise_resend:
+      promise_resend: Nous vous enverrons un email avec le lien de confirmation pour activer pour votre compte si vous en avez un.
     new:
-      submit:
-      title:
-      will_email_notice:
+      submit: Renvoyer la confirmation
+      title: Renvoyer l'email de confirmation
+      will_email_notice: Nous vous enverrons un email avec le lien de confirmation pour activer pour votre compte.
```

Comment:
I went from https://github.com/rubygems/rubygems.org/blob/master/config/locales/en.yml#L53 and used some inspiration from https://github.com/tigrish/devise-i18n/blob/master/rails/locales/fr.yml#L36.

I suggest "Si ce compte existe, vous recevrez un email avec un lien de confirmation pour l'activer."